Revert/Sources/HomeViewController.swift

```diff
@@ -63,8 +59,7 @@ final class HomeViewController: UITableViewController {
   private var collection = CollectableCollection<HomeItem>(items: .Home)
   private var dataSource: DataSource<HomeItem, HomeCell>
   private var isSplitViewControllerCollapsed: Bool {
-    // If self.splitViewController is nil then we're on iOS7 on phone, i.e equivalent to collapsed mode
-    return self.splitViewController?.backwardCompatibleCollapsed ?? true
+    return self.splitViewController?.collapsed ?? true
   }
```

Comment:
Well I reckon the `splitViewController` will always be present on iOS8+ as that's when the 6+ was unveiled.

So technically it should never be `nil`.. Are we happy to rely on this detail implementation or do we rather want this to be as explicit as possible?